filelock/filelock.go

```diff
@@ -1,3 +1,4 @@
+//go:build !solaris
 // +build !solaris
```

Comment:
Should we only keep the newer//more readable version here ?

Reply:
Maybe not in case this gets built with go 1.16

Reply:
Oh interesting. I thought it was a direct replace of the build tag. Packer currently has both so maybe we can keep it consistent, and remove once 1.18 is out. What do you think?

Reply:
I'm merging as is for now. During our sync tomorrow we can make a call to add or remove here and in Packer if we don't want to wait until Go 1.18.

Reply:
yup sounds good to me !